Doc/library/curses.rst

```diff
@@ -1294,6 +1296,9 @@ Several constants are available to specify character cell attributes:
 | ``A_UNDERLINE``  | Underline mode.               |
 +------------------+-------------------------------+
 
+.. versionadded:: 3.7
+    ``A_ITALIC`` was added.
```

Comment:
`A_ITALIC` is an extension so it maybe not supported by all platforms? So maybe this sentence should be _``A_ITALIC`` is supported if the underlying curses library support it_? @berkerpeksag how do you think?